VSTS/Tests/L0/SonarQubeScannerMsBuildEnd/TopLevelOrchestration.ps1

```diff
@@ -11,6 +11,8 @@ param()
 Register-Mock InvokeMSBuildRunnerPostTest 
 Register-Mock CreateAndUploadReport 
 Register-Mock BreakBuildOnQualityGateFailure 
+Register-Mock IsFeatureEnabled {$true}
+Register-Mock HandleCodeAnalysisReporting
 
 # Act
 . $PSScriptRoot\..\..\..\Tasks\SonarQubeScannerMsBuildEnd\SonarQubePostTest.ps1    
```

Comment:
Not sure why this works outside Pester, possibly there's extra mocking of Get-TaskVariable.